target/bin/listmailuser

```diff
@@ -3,10 +3,39 @@
 # shellcheck source=../scripts/helper-functions.sh
 . /usr/local/bin/helper-functions.sh
 
-DATABASE=${DATABASE:-/tmp/docker-mailserver/postfix-accounts.cf}
+function dovecot_quota_to_hr()
+{
+  if [ "${1}" == "-" ]
+  then
+    echo "~"
+  else
+    echo $(( 1024 * ${1} )) | numfmt --to=iec
+  fi
+}
 
-[[ -f ${DATABASE} ]] || errex "Error: No postfix-virtual.cf file"
-[[ -s ${DATABASE} ]] || errex "Error: Empty postfix-virtual.cf - no aliases have been added"
+DATABASE="/tmp/docker-mailserver/postfix-accounts.cf"
+ALIASES="/tmp/docker-mailserver/postfix-virtual.cf"
+
+[[ -f ${DATABASE} ]] || errex "Error: No postfix-accounts.cf file"
+[[ -s ${DATABASE} ]] || errex "Error: Empty postfix-accounts.cf - no accounts have been added"
+
+while read -r LINE
+do
+  USER=$(echo "${LINE}" | cut -d'|' -f1)
+
+  # ${QUOTA[0]} => current size
+  # ${QUOTA[1]} => configured size limit
+  # ${QUOTA[2]} => usage in percent
+  IFS=' ' read -r -a QUOTA <<< "$(doveadm quota get -u "${USER}" | tail +2 | awk '{ if ($3 == "STORAGE") { print $4" "$5" "$6 } }')"
+
+  echo "* ${USER} ( $(dovecot_quota_to_hr "${QUOTA[0]}") / $(dovecot_quota_to_hr "${QUOTA[1]}") ) [${QUOTA[2]}%]"
+
+  if [[ -f ${ALIASES} ]] && grep -q "${USER}" "${ALIASES}"
+  then
+    echo -e "    [ aliases -> $(grep "${USER}" "${ALIASES}" | awk '{print $1;}' | sed ':a;N;$!ba;s/\n/, /g')]\n"
```

Comment:
Worth adding a hint that the `sed ...` replacing newlines with a comma?

Just bc for me it was not clear on a first sight and as you mentioned in the Issue you needed some time too ;) 

Thinking of: 
> [x] I have commented my code, particularly in hard-to-understand areas

Other than that: LGTM 👍🏻

Reply:
I must say, the beginning of the `sed` is something I have not seen either, but I don't feel like this needs a comment, as it becomes clear that `.../\n/, /g` replaces newlines :D This is just my PoV, feel free to add it nevertheless if you feel like it's really valuable.

Reply:
For future reference: https://stackoverflow.com/a/1252191/568737

Reply:
I only knew the shell-escape pattern with `$'\n'`, but this is interesting as well.